challenges/project_management.md

```diff
@@ -0,0 +1,21 @@
+# Project Management
+
+## Timesheets
+
+- Timesheets are required by some funders (I.e UKRI).
+- Timesheets are required by TRAC methodology.
+- Timesheets can be used to compare expected FTE on a project with actual FTE.
+- There is an overhead to completing timesheets.
+- There can sometimes be a discrepency between actual time spent and the time
+  recorded on the timesheet to be submitted to the funder.
```

Comment:
This should be avoided since it contravenes the terms of the grant agreement. Albeit minor approximation is unlikely to be an issue.